include/exceptions.hh

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <string>
+#include <stdexcept>
+
+#include "cassandra.h"
+
+namespace scmd {
```

Comment:
Indentation in namespaces? Sure, you can do as you wish, but let's be consistent throughout the repo.